docs/.vuepress/components/WrapLayoutLiveDemoDoc.vue

```diff
@@ -0,0 +1,344 @@
+            id="wraplayout_addNewChild__button"
+          >Add New Child</button>
+          <button
+            class="btn m-g-3"
+            @click="removeLabels()"
+            id="wraplayout_removeAll__button"
+          >Remove Children</button>
+          <button
+            class="btn m-g-3"
+            @click="removeLast()"
+            id="wraplayout_removeLast__button"
+          >Remove Last Child</button>
+        </StackLayout>
+        <StackLayout orientation="horizontal" class="m-t-5">
+          <Label text="Text: " width="140"></Label>
+          <TextField
+            v-model="itemText"
+            @returnPress="onKeyUp"
+            keyboardType="text"
+            ref="itemInput"
+            hint="Item Text"
+            id="wraplayout_itemText_input"
+          ></TextField>
+        </StackLayout>
+        <StackLayout orientation="horizontal" class="m-t-5">
+          <Label text="Width: " width="140"></Label>
+          <TextField
+            keyboardType="number"
+            v-model="itemWidth"
+            hint="Item Width"
+            id="wraplayout_innerItem_itemWidth_input"
+          ></TextField>
+        </StackLayout>
+        <StackLayout orientation="horizontal" class="m-t-5">
+          <Label text="Height: " width="140"></Label>
+          <TextField
+            keyboardType="number"
+            v-model="itemHeight"
+            hint="Item Height"
+            id="wraplayout_innerItem_itemHeight_input"
+          ></TextField>
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+
+    <StackLayout class="demo-panel">
+      <Label class="demo-panel__title" text="Children List"></Label>
+      <StackLayout class="form-group">
+        <StackLayout
+          orientation="horizontal"
+          v-for="(label,index) in labels"
+          :key="index"
+          class="m-t-5"
+        >
+          <Label :text="(index +1) + '.'" width="30"></Label>
+
+          <Label text="Text :" class="m-g-3"></Label>
+          <TextField
+            v-model="label.text"
+            width="100"
+            :id="'wraplayout_childreneditor_text_input' + index"
+          ></TextField>
+
+          <Label text="Width: " class="m-g-3"></Label>
+          <TextField
+            keyboardType="number"
+            v-model="label.width"
+            width="100"
+            :id="'wraplayout_childreneditor_width_input' + index"
+          ></TextField>
+
+          <Label text="Height: " class="m-g-3"></Label>
+          <TextField
+            keyboardType="number"
+            v-model="label.height"
+            width="100"
+            :id="'wraplayout_childreneditor_height_input' + index"
+          ></TextField>
+
+          <Label text="Color: " class="m-g-3"></Label>
+          <input
+            type="color"
+            v-model="label.bgColor"
+            style="width:100px"
+            :id="'wraplayout_childreneditor_bgColor_input' + index"
+          >
+
+          <i
```

Comment:
lets use button component instead of <i> tag

Reply:
done